studio/components/interfaces/Auth/Policies/Policies.tsx

```diff
@@ -0,0 +1,213 @@
+    setSelectedPolicyToEdit(policy)
+    setSelectedSchemaAndTable({ schema: policy.schema, table: policy.table })
+  }
+
+  const onSelectDeletePolicy = (policy: any) => {
+    setSelectedPolicyToDelete(policy)
+  }
+
+  const onSavePolicySuccess = async () => {
+    ui.setNotification({ category: 'success', message: 'Policy successfully saved!' })
+    closePolicyEditorModal()
+  }
+
+  // Methods that involve some API
+  const onToggleRLS = async () => {
+    const payload = {
+      id: selectedTableToToggleRLS.id,
+      rls_enabled: !selectedTableToToggleRLS.rls_enabled,
+    }
+
+    const res: any = await meta.tables.update(payload.id, payload)
+    if (res.error) {
+      ui.setNotification({
+        category: 'error',
+        message: `Failed to toggle RLS: ${res.error.message}`,
+      })
+    }
+    closeConfirmModal()
+  }
+
+  const onCreatePolicy = async (payload: any) => {
+    const res = await meta.policies.create(payload)
+    if (res.error) {
+      ui.setNotification({
+        category: 'error',
+        message: `Error adding policy: ${res.error.message}`,
+      })
+      return true
+    }
+    return false
+  }
+
+  const onUpdatePolicy = async (payload: any) => {
+    const res = await meta.policies.update(payload.id, payload)
+    if (res.error) {
+      ui.setNotification({
+        category: 'error',
+        message: `Error updating policy: ${res.error.message}`,
+      })
+      return true
+    }
+    return false
+  }
+
+  const onDeletePolicy = async () => {
+    const res = await meta.policies.del(selectedPolicyToDelete.id)
+    if (typeof res !== 'boolean' && res.error) {
+      ui.setNotification({
+        category: 'error',
+        message: `Error deleting policy: ${res.error.message}`,
+      })
+    } else {
+      ui.setNotification({ category: 'success', message: 'Successfully deleted policy!' })
+    }
+    closeConfirmModal()
+  }
+
+  return (
+    <>
+      {tables.length > 0 ? (
+        tables.map((table: any) => (
+          <section key={table.id}>
+            <PolicyTableRow
+              table={table}
+              isLocked={isLocked}
+              onSelectToggleRLS={onSelectToggleRLS}
+              onSelectCreatePolicy={onSelectCreatePolicy}
+              onSelectEditPolicy={onSelectEditPolicy}
+              onSelectDeletePolicy={onSelectDeletePolicy}
+            />
+          </section>
+        ))
+      ) : hasTables ? (
+        <NoSearchResults />
+      ) : (
+        <div className="flex-grow">
+          <ProductEmptyState
+            size="large"
+            title="Postgres Policies"
+            ctaButtonLabel="Create a table"
+            infoButtonLabel="What is RLS?"
+            infoButtonUrl="https://supabase.com/docs/guides/auth/row-level-security"
+            onClickCta={() => router.push(`/project/${ref}/editor`)}
+          >
+            <div className="space-y-4">
+              <InformationBox
+                title="What are policies?"
```

Comment:
There's some consistency things here I think — "Postgres policies", "policies", "RLS Policies", "security policy" 

Can we always do "RLS policy"  / "Postgres RLS policy"